Src/BootCamp.Chapter/FileCleaner.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.IO;
+
+namespace BootCamp.Chapter
+{
+    public static class FileCleaner
+    {
+        public static void Clean(string inputFile, string outputFile)
+        {
+            if (String.IsNullOrEmpty(inputFile) || String.IsNullOrEmpty(outputFile))
+            {
+                throw new ArgumentException("File path is null or empty.");
+            }
+
+            var corruptedFileContent = File.ReadAllText(inputFile);
+
+            if (IsFileClean(corruptedFileContent) || String.IsNullOrEmpty(corruptedFileContent))
+            {
+                File.WriteAllText(outputFile, corruptedFileContent);
+                return;
+            }
+            
+            if (String.IsNullOrEmpty(corruptedFileContent))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change this condition so that it does not always evaluate to 'false'; some subsequent code is never executed.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5070129)